### PR TITLE
TMEDIA-907: Convert more source blocks to use new resizer

### PR DESCRIPTION
**Pull request by nschubach**
## Description

Convert the following content sources to use image resizer V2:
* story-feed-author-content-source-block
* story-feed-query-content-source-block
* story-feed-sections-content-source-block
* story-feed-tag-content-source-block

## Jira Ticket

- [TMEDIA-907](https://arcpublishing.atlassian.net/browse/TMEDIA-907)

## Acceptance Criteria

Convert the following content sources to use image resizer V2:
* story-feed-author-content-source-block
* story-feed-query-content-source-block
* story-feed-sections-content-source-block
* story-feed-tag-content-source-block

## Test Steps

1. Checkout this branch `git checkout tmedia-907`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/story-feed-author-content-source-block,@wpmedia/story-feed-query-content-source-block,@wpmedia/story-feed-sections-content-source-block,@wpmedia/story-feed-tag-content-source-block`
3. http://localhost/pagebuilder/tools/debugger?_website=themesinternal&type=content-debugger&source=story-feed-author&config={%22authorSlug%22:%22tariq-alshwaiki%22}
4. Inspect the data for `auth-tokens` (content-elements[0].content-elements[4])
5. http://localhost/pagebuilder/tools/debugger?_website=themesinternal&type=content-debugger&source=story-feed-sections&config={%22excludeSections%22:%22%22,%22includeSections%22:%22/demo%22}
6. http://localhost/pagebuilder/tools/debugger?_website=themesinternal&type=content-debugger&source=story-feed-tag&config={%22tagSlug%22:%22dogs%22}
7. http://localhost/pagebuilder/tools/debugger?_website=themesinternal&type=content-debugger&source=story-feed-query

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/story-feed-author-content-source-block/package.json [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
-  "peerDependencies": {
-    "@wpmedia/resizer-image-block": "*"
-  }
+	"name": "@wpmedia/story-feed-author-content-source-block",
+	"version": "5.14.1",
+	"description": "Content source block for story feed queries by author",
+	"author": "Arc XP - Themes",
+	"homepage": "https://github.com/WPMedia/arc-themes-blocks",
+	"license": "CC-BY-NC-ND-4.0",
+	"main": "index.js",
```
Comment (matthewroach) [inline blocks/story-feed-author-content-source-block/package.json]: We can remove the `index.js` from main as we do not support importing these

Comment (matthewroach) [inline blocks/story-feed-author-content-source-block/package.json]:
```suggestion
	"main": "",
```

File: blocks/story-feed-author-content-source-block/package.json [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+		"access": "public"
+	},
+	"repository": {
+		"type": "git",
+		"url": "ssh://git@github.com/WPMedia/arc-themes-blocks.git",
+		"directory": "blocks/story-feed-author-content-source-block"
+	},
+	"gitHead": "83fb8d6685958d85ddbfcfbf01d9a5c864cd6c95",
+	"peerDependencies": {
+		"@wpmedia/resizer-image-block": "*",
```
Comment (matthewroach) [inline blocks/story-feed-author-content-source-block/package.json]: We shouldn't have a dependency on the `resizer-image-block` with these changes
```suggestion

```

**Review by matthewroach — CHANGES_REQUESTED** [review]
Couple small changes

File: blocks/story-feed-author-content-source-block/package.json (hunk 2 above)
Comment (nschubach) [reply blocks/story-feed-author-content-source-block/package.json]: Whoops, yeah, that was a copy/paste issue.

File: blocks/story-feed-author-content-source-block/package.json (hunk 1 above)
Comment (nschubach) [reply blocks/story-feed-author-content-source-block/package.json]: Gotcha

Comment (nschubach) [reply blocks/story-feed-author-content-source-block/package.json]: Will remove

**Review by matthewroach — APPROVED** [review]

**Pull request merged**